src/lib/y2storage/bcache_cset.rb

```diff
@@ -0,0 +1,61 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/storage_class_wrapper"
+
+module Y2Storage
+  # A Bcache caching set
+  #
+  # This is a wrapper for Storage::BcacheCset
+  class BcacheCset < Device
+    wrap_class Storage::BcacheCset
+
+    # @!method blk_device
+    #   @return [BlkDevice] returns a backing device for cache
+    storage_forward :blk_devices, as: "BlkDevice"
```

Comment:
Is `:blk_devices` or `:blk_device`?

Reply:
Probably the declaration is right but the documentation is a copy&paste from `Bcache`. Isn't it?

Reply:
yes, will fix it.